specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2018-02-14-preview/keyvault.json

```diff
@@ -877,6 +877,11 @@
           "type": "boolean",
           "description": "Property to specify whether the 'soft delete' functionality is enabled for this key vault. It does not accept false value."
         },
+        "softDeleteRetentionInDays": {
```

Comment:
Thank you fixing the data plane, but this management plane already shipped as well. I see a stable/2018-02-14, which means you shouldn't be modifying a preview for this version either. /cc @johanste 

OpenAPI Hub can be used to create new versions for the management plane as well, like I did for the data plane.